docker/Dockerfile.application

```diff
@@ -16,14 +16,19 @@ RUN apk add --update curl tzdata iproute2 bash libc6-compat vim &&  \
  	cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime && \
  	echo "Asia/Shanghai" >  /etc/timezone
 
-RUN curl -o helm-v3.0.3-linux-amd64.tar.gz http://aliacs-k8s-cn-hongkong.oss-cn-hongkong.aliyuncs.com/public/pkg/helm/helm-v3.0.3-linux-amd64.tar.gz && \
+RUN if [ `uname -m` == "aarch64" ] ; then curl -o helm-v3.0.3-linux-arm64.tar.gz https://get.helm.sh/helm-v3.0.3-linux-arm64.tar.gz && \
```

Comment:
Maybe you can use `TARGETARCH` instead of run `uname -m` in dockerfile, refer to https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope.

An example for reference: https://github.com/juicedata/juicefs-csi-driver/blob/master/Dockerfile#L41

Reply:
TARGETARCH works well while using docker buildx build. But normal docker build fails to detect TARGETARCH in the RUN clause and thus the helm tar file in the Dockerfiles isn’t getting curled. I have observed the same on both ARM64 and AMD64 platforms with docker build. 
~~~
Step 4/4 : RUN curl -o helm-v3.0.3-linux-${TARGETARCH}.tar.gz https://get.helm.sh/helm-v3.0.3-linux-${TARGETARCH}.tar.gz &&     tar -xvf helm-v3.0.3-linux-${TARGETARCH}.tar.gz &&     mv linux-${TARGETARCH}/helm /usr/local/bin/ddc-helm &&     chmod u+x /usr/local/bin/ddc-helm &&     rm -f helm-v3.0.3-linux-${TARGETARCH}.tar.gz
 ---> Running in 4aeeac5ded68
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   215  100   215    0     0    817      0 --:--:-- --:--:-- --:--:--   817
tar: invalid tar magic
The command '/bin/sh -c curl -o helm-v3.0.3-linux-${TARGETARCH}.tar.gz https://get.helm.sh/helm-v3.0.3-linux-${TARGETARCH}.tar.gz &&     tar -xvf helm-v3.0.3-linux-${TARGETARCH}.tar.gz &&     mv linux-${TARGETARCH}/helm /usr/local/bin/ddc-helm &&     chmod u+x /usr/local/bin/ddc-helm &&     rm -f helm-v3.0.3-linux-${TARGETARCH}.tar.gz' returned a non-zero code: 1
~~~

Reply:
add `ARG TARGETARCH` in dockerfile and have a try? It works well in this way.

Reply:
I have already used `ARG TARGETARCH` in my Dockerfiles but was still facing the same issue.

Actually, in docker build, we need to specify flag `--build-arg TARGETARCH=amd64` along with docker build command to use TARGETARCH in Dockerfile.

`$ sudo docker build --build-arg TARGETARCH=amd64 -f docker/Dockerfile.application -t <NAME> .
`


I will amend the PR to use `--build-arg` flag with docker build commands.

Reply:
Misunderstood. I think we can explicitly add `--build-arg` in docker build commands and make dockerfile more clear. What do you think @cheyang ?

Reply:
Sure. I also prefer `--build-arg` to make the dockerfile easy to understand.